berkeley-mobile/Fitness/Fitness+Controllers/GymClassesController.swift

```diff
@@ -62,8 +62,34 @@ extension GymClassesController: UITableViewDataSource {
             cell.eventCategory.backgroundColor = gymClass.color
             cell.eventCategory.isHidden = gymClass.class_type == nil
             cell.eventTaggingColor.backgroundColor = gymClass.color
+            
+            if gymClass.website_link != nil && gymClass.location == "Zoom" {
+                let tapGesture = EventTapGestureRecognizer(target: self, action:
+                                                            #selector(GymClassesController.zoomTapped(gesture:)))
+                tapGesture.eventUrl = gymClass.website_link!
+                
+                cell.cellSetImage(image: UIImage(named: "Zoom Logo")!,
+                                  tapGesture: tapGesture)
+            }
         }
         return cell
     }
     
+    @objc func zoomTapped(gesture: EventTapGestureRecognizer) {
+        if (gesture.view as? UIImageView) != nil {
+            vc.presentAlertWithTitle(title: "Are you sure you want to open Zoom?", message: "Berkeley Mobile wants to open a online fitness class in Zoom", options: "Cancel", "Yes") { (option) in
```

Comment:
"Berkeley Mobile wants to open a online fitness class in Zoom" - an instead of a

Reply:
also not sure if we want to say "Are you sure you want to" Could change to something like "Would you like to" or ask design about the message we want